docs/timeline/index.html

```diff
@@ -1124,13 +1185,6 @@ <h2 id="Configuration_Options">Configuration Options</h2>
       This would be used as an additional way to add content that is constant in size with the visible frame of the item and does not get visibly hidden with the item's internal container: <code>vis-item-overflow</code> which is <code>overflow:hidden</code>.</td>
     </tr>
 
-    <tr class='deprecated'>
-      <td>throttleRedraw</td>
```

Comment:
@yotamberk We should check if that really has been removed.

Reply:
It has. I removed it.